src/main/java/cs321/btree/BTree.java

```diff
@@ -144,32 +206,14 @@ private boolean writeDisk(BTreeNode node) {
             oos.close();
             fos.close();
 
-            // System.out.println(String.format("[%s] -> %d bytes", f.getAbsolutePath(), f.length()));
+            System.out.println(String.format("[%s] -> %d bytes", f.getAbsolutePath(), f.length()));
         } catch (Exception e) {
             return false;
         }
 
         return true;
     }
 
-    /**
-     * Update the main meta file to store the new core data such as the root node, size, etc.
-     */
-    private void updateMetaFile() {
```

Comment:
Why was this deleted? This needs to be in place to update the current size and root node of the Tree